<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>687</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Montserrat Medium</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Aleatorizador de Tareas</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>30</number>
    </property>
    <property name="rightMargin">
     <number>30</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>10</number>
    </property>
    <item row="6" column="0">
     <widget class="QLabel" name="tarea_input_label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nombre de la tarea: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QPushButton" name="boton_aleatorio">
      <property name="text">
       <string>Aleatorizar</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="borrar_tarea_input">
      <property name="maxLength">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="boton_agregar_tarea">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Agregar tarea</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="borrar_tarea_input_label">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingrese el ID de la tarea a borrar:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="boton_quitar_tarea">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Quitar tarea</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="tarea_input"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="subtitulo2">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Escribe tus tareas y yo eligiré por ti!!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="subtitulo">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>¿No sabe por cuál tarea comenzar?</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLabel" name="table_titulo">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tareas:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="titulo">
      <property name="font">
       <font>
        <family>Montserrat Medium</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bienvenido!</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QListWidget" name="tareas">
      <property name="font">
       <font>
        <family>Montserrat</family>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
